pkg/execution/state/redis_state/redis_state_test.go

```diff
@@ -167,6 +167,7 @@ func BenchmarkNew(b *testing.B) {
 	r := miniredis.RunT(b)
 	sm, err := New(
 		context.Background(),
+		WithKeyGenerator(DefaultKeyFunc{Prefix: "{state}"}),
```

Comment:
what's this for?

Reply:
making sure tests work in case you hook it up to a clustered redis for unit tests.